### ISSUE TITLE
Specyfying name for stores does not work

### ISSUE THREAD

**Issue by tbitowt** [zalmoxisus/redux-devtools-extension#212]
I have problem with specyfing names for the stores which should be visible in devTools window.

I'm using `react-redux-provide` in my project and I've configured my project actually to work with devtools-extension, but the problem is that instead of store name, extension shows me some kind of hashcode.

I have done steps:

- added this to the place where store is created

`
window.devToolsExtension ? window.devToolsExtension({name: storeKey}) : f => f
` 
- logged `storeKey` to be sure that it's name is proper
- run application

in the console log i can see nice names for my stores, but in devTools extension I see:
![image](https://cloud.githubusercontent.com/assets/6546880/18582295/981730ee-7c04-11e6-8d68-9f396a63b557.png)

There are proper number of stores, but it does not take a name to display.

I have reinstalled extension on my browser, but this didn't help.

P.S.
When I remove the registration line the stores does not shows up in the extension, so it isn't a problem with that probably.

//EDIT

2 more things:
- It's strange that devTools extensions shows in main browser window automatically with instance selector. On other computers it shows without that selector.
- When I reload page while devTools extension is visible. It's loosing connection to stores, and shows message:
`
No store found. Make sure to follow the instructions.
`

**Comment by tbitowt** [comment]
2 more things:
- It's strange that devTools extensions shows in main browser window automatically with instance selector. On other computers it shows without that selector.
- When I reload page while devTools extension is visible. It's loosing connection to stores, and shows message:
`
No store found. Make sure to follow the instructions.
`

**Comment by zalmoxisus** [comment]
Thanks for the report. I cannot replicate that problem with [our examples](https://github.com/zalmoxisus/redux-devtools-extension/tree/master/examples), adding a `name` parameter works as expected. An example to replicate that would be welcome (probably forking and changing the examples from [`react-redux-provide`](https://github.com/loggur/react-redux-provide)) as I'm not familiar with that library.

> It's strange that devTools extensions shows in main browser window automatically with instance selector. On other computers it shows without that selector.

It was introduced in [the yesterday update](https://github.com/zalmoxisus/redux-devtools-extension/releases/tag/v2.6.0), so probably on other computers you're still using the old version.

> When I reload page while devTools extension is visible. It's loosing connection to stores, and shows message:  `No store found. Make sure to follow the instructions.`

I can replicate this one, will fix asap. A temporarily workaround will be to open the extensions popup or window to update it.

**Comment by tbitowt** [comment]
@zalmoxisus: It seems that yesterday update broke the naming  of stores too. It seems that it's working fine with version 2.5.1.9.

**Comment by zalmoxisus** [comment]
Well, before we were sending `name` with every update, which wasn't optimal. Now we're sending it only with first update.

I need an example to reproduce the problem to see how we can handle that use case. I've checked for Redux, React and Mobx. For all 3 examples the name parameter is passed properly.

**Comment by tbitowt** [comment]
Here's my reproduction package.
`reproduction.zip` contains my source files
react-redux-provide.zip is a copy of node_module with my changes (only added devtools_extension registration in `createProviderStore.js` file)
[react-redux-provide.zip](https://github.com/zalmoxisus/redux-devtools-extension/files/476758/react-redux-provide.zip)
[reproduction.zip](https://github.com/zalmoxisus/redux-devtools-extension/files/476759/reproduction.zip)

**Comment by zalmoxisus** [comment]
@tbitowt, thanks! I replicated and fixed it. Will publish on Chrome Store in few hours.

BTW, you can [pass Extension's enhancer to `react-redux-provide`](https://github.com/loggur/react-redux-provide#enhancer) without forking it.

**Comment by tbitowt** [comment]
Thanks,

BTW. Thanks for finding the solution for not forking this library!

**Comment by zalmoxisus** [comment]
All the issues should be fixed in `2.6.1`. If there still something goes wrong, feel free to reopen the issue.

**Comment by stefk** [comment]
Hi,

`2.6.1` did fix the *no store found* error for me, but now there aren't any actions shown after `@@INIT` anymore. I'm almost 100% sure it isn't a mutation issue as I use redux-freeze. Also the same code base was working fine before `2.6.0`. Should I open a new issue?

This is really blocking, so any feedback would be greatly appreciated!

**Comment by stefk** [comment]
It appears that sometimes simply upgrading from chrome://extensions isn't enough. After restarting the browser, everything works fine. Sorry for the noise.

**Comment by zalmoxisus** [comment]
@stefk, no probs. If you're experiencing this problem again, please let me know, check "Developer mode" on top of `chrome://extensions` page, and then click on `background page` bellow the extension's name. There should be an error message.

**Comment by levonlin** [comment]
Hi @zalmoxisus, I use the 2.6.1 version, but sometimes the "no store found error" would appear in the chrome DevTool panel, and if I open this redux devtool extension with pop windows, it always works ok. I have checked the error information with the [TodoMVC demo](http://zalmoxisus.github.io/examples/todomvc/), here is the log: 
```
Error in event handler for (unknown): Error: Attempting to use a disconnected port object
    at chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/background.bundle.js:3:11980
    at o (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/background.bundle.js:3:11949)
    at r (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/background.bundle.js:3:13441)
Error in event handler for (unknown): Error: Attempting to use a disconnected port object
    at chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/background.bundle.js:3:11980
    at o (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/background.bundle.js:3:11949)
    at u (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/background.bundle.js:3:13331)
    at n (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/background.bundle.js:3:13844)
```

**Comment by zalmoxisus** [comment]
@levonlin, should be fixed in `2.6.1.1`.
